Realtime-josh/EPIC_Mail
# Bug(UI)

**What Does This PR Do?**
This task is a bug. Some files found in UI pages are irrelevant and should be removed.This PR ensures that files and folders should contain only relevant documents that are to be linked up together accordingly. Make sure UI is responsive and there are non cluttered files found inside directory.

How Should This Be Manually Tested?
Follow URL: https://github.com/Realtime-josh/EPIC_Mail. Please make sure to select gh-pages as the preferred branch found on the drop-down located on github. Clone repository and ensure that files are being retrieved on local device. All files and folders should contain useful elements.

[PT Story ID- 164680219]
